Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml

```diff
@@ -603,6 +605,15 @@ Item {
 
         if (leaderPosition === Enums.LeaderPosition.Automatic) {
 
+            // Move leader vertically if vertical position isn't optimal
```

Comment:
@nandinirao I tested all the positions on Android and iOS Simulator. Android looks good but there are some issues with iOS where I can't test very well. I chatted with @ldanzinger and that's a known issue right now. It was certainly no worse on iOS.